ellipsoid arena
# ellipsoid[] arena_collision
ellipsoid[] obstacles
int64 obstacle_size
ellipsoid robot